requirements.txt

```diff
@@ -0,0 +1,3 @@
+python-glanceclient==0.18.0
+python-keystoneclient==1.4.0
+
```

Comment:
Due to you are not using the latest versions of that libs, I think you should add the **oslo.** lib dependencies for avoiding *import* errors when executing operations from that libs. We have had problems in FiHelath consecuence of that: **CLAUDIA-5316**